index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body>
+    <p> test gps in body </p>
+    <a href="https://developers.google.com/maps/documentation/javascript/overview#javascript"> link to new api</a>
+    <div id="divId"></div>
+    <div id="map">
+
+</div>
+</body>
+
+<script>
```

Comment:
If you want to make something more advanced than "Hello world" please make sure it is working as expected, becase in current implementation I see this 
![image](https://github.com/MolchanovAlexander/hz_29_01_2024/assets/69054906/02404cf1-06aa-4c78-9d1b-06ac971ff905)